src/qinst/instrument.py

```diff
@@ -56,3 +58,18 @@ def get(self, *args) -> dict:
             else:
                 raise RuntimeError(f"The instrument does not have the {key} parameter.")
         return values
+
+    def validate_opt(self, opt: str, allowed: tuple):
+        """Check if provided option is between allowed ones."""
+        if opt not in allowed:
+            raise RuntimeError(f"Invalid option provided, choose between {allowed}")
+
+    def validate_range(self, n, n_min, n_max):
```

Comment:
Since `self` is not used, this can be a staticmethod:
```suggestion
    @staticmethod
    def validate_range(self, n, n_min, n_max):
```